src/core/amp-element.extern.js

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
It's not clear to me if this belongs in core, and if it does I think we should move it when we move amp-element.js itself into core